analysis/analysis.py

```diff
@@ -92,6 +93,38 @@ def run_analysis():
     show_response_groups(response_groups_freq_dicts)
     total_view_time = compute_total_view_time(student_data)
     print(f'The total view time of all students was {total_view_time}.')
+    print(count_revists(student_data))
+
+
+def count_revists(data):
```

Comment:
this function name is misleading -- we're not counting revisits, we're getting the mean count.

Reply:
Changed it to `compute_mean_revisits(data)`